client/config_parser.py

```diff
@@ -92,10 +96,14 @@ def parse_config(self, config_file: str = 'config.yaml') -> TranslationConfig:
         translation_config_input = data[self.__TRANSLATION_CONFIG]
         config.location = translation_config_input["location"]
         config.translation_type = translation_config_input[self.__TRANSLATION_TYPE]
-        config.input_directory = self.__DEFAULT_INPUT_DIR if self.__INPUT_DIR not in translation_config_input \
+        input_directory = self.__DEFAULT_INPUT_DIR if self.__INPUT_DIR not in translation_config_input \
             else translation_config_input[self.__INPUT_DIR]
+        config.input_directory = validate_path(input_directory)
         config.output_directory = self.__DEFAULT_OUTPUT_DIR if self.__OUTPUT_DIR not in translation_config_input \
             else translation_config_input[self.__OUTPUT_DIR]
+        if os.getcwd().endswith(REPO_ROOT_DIR) and config.output_directory == self.__DEFAULT_OUTPUT_DIR:
+            config.output_directory = join(CLIENT_DIR, self.__DEFAULT_OUTPUT_DIR)
```

Comment:
Agreed.  I updated this part so that users can run the tool from any directories.  If cwd is not the root, user should specify the path info through args. Otherwise, there will be warnings and the follow-up validations will likely to fail.